mkdocs.yml

```diff
@@ -54,7 +54,8 @@ nav:
         - 'Post-Root Procedures': 'Hacking/PostRoot.md'
     - 'Firmware Repository': 'Repository.md'
     - 'Other guides':
-      - 'Safe Firmware Upgrade': 'Upgrade.md'
+      - 'Safe RBI Firmware Upgrade': 'Upgrade.md'

```

Comment:
Mentioning RBI in the page tilte could be confusing in case someone would flash a raw bank dump. If we really want to use RBI and PKGTB names to refer firmwares that could have been packaged in those formats but are actually flashable from unpacked dumps then we shall rename traditional bank dumps to "squashfs bank dumps", coherently with "ubifs bank dumps" you used in Upgrade-pkgtb.md page

Reply:
I have reverted the RBI change, and modified the Upgrade-pkgtb.md title to "Safe Firmware Upgrade (U-boot bootloader).

I have also added warnings into Upgrade.md and Upgrade-pkgtb.md to use the other procedure for the relevant device config/firmware file.